packages/blade/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "baseUrl": ".",
     "paths": {
       "~utils": ["src/utils"],
+      "~utils/*": ["src/utils/*"],
```

Comment:
is this separately required? because we've been importing utilities with `~utils/something` always and it did work earlier 🤔

Reply:
Saurabh, I tried to check do it otherwise, typescript is complaining without this change. Please let me know if I missed anything. Example - `Cannot find module '~utils/makeTypographySize' or its corresponding type declarations`

Reply:
oh right earlier we were exporting them as `~src/utils/makeTypographySize` 🤦🏼 makes sense